docs/api-table.js

```diff
@@ -814,10 +815,10 @@ export const vue = {
       [
         'placement', "'top-start' | 'top' | 'top-end' | 'bottom-start' | 'bottom' | 'bottom-end' | 'left-start' | 'left' | 'left-end' | 'right-start' | 'right' | 'right-end'", "'bottom'", 'Placement according to the target element. The arrow will point to the opposite side of this position. Replaces props `bottom`, `top`, `left`, `right`',
       ],
-      ['bottom', 'boolean', 'false', '@deprecated use `placement` instead'],
-      ['top', 'boolean', 'false', '@deprecated use `placement` instead'],
-      ['left', 'boolean', 'false', '@deprecated use `placement` instead'],
-      ['right', 'boolean', 'false', '@deprecated use `placement` instead'],
+      ['~~bottom~~', 'boolean', 'false', '@deprecated use `placement` instead'],
+      ['~~top~~', 'boolean', 'false', '@deprecated use `placement` instead'],
+      ['~~left~~', 'boolean', 'false', '@deprecated use `placement` instead'],
+      ['~~right~~', 'boolean', 'false', '@deprecated use `placement` instead'],
```

Comment:
I believe these props are also deleted from Attention component. The message that they were deprecated was already there for some time, so I'd say it's time to remove them from the docs, too.

Reply:
Removed in this commit: [e50fc86](https://github.com/warp-ds/tech-docs/pull/216/commits/e50fc86b42e9a6d7f35299f66c7385049136b84a)